storage/mysql/storage_test.go

```diff
@@ -240,11 +244,9 @@ func cleanTestDB(db *sql.DB) {
 	}
 }
 
-func createFakeSignedLogRoot(db *sql.DB, tree *trillian.Tree, treeSize uint64) {
+func createFakeSignedLogRoot(ctx context.Context, t *testing.T, l storage.LogStorage, tree *trillian.Tree, treeSize uint64) {
```

Comment:
Pre-existing but would be better if named `signRootOrDie` or whatever matches rename of fn below.